OneTrustConsentFilter/gradle.properties

```diff
@@ -20,4 +20,19 @@ kotlin.code.style=official
 # Enables namespacing of each library's R class so that its R class includes only the
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
-android.nonTransitiveRClass=true
\ No newline at end of file
+android.nonTransitiveRClass=true
+android.enableJetifier=true
+POM_LICENCE_URL=http://opensource.org/licenses/MIT
+POM_DEVELOPER_ID=Rudderstack
+POM_NAME=Rudder-Onetrust-Consent-Interceptor-Android
+POM_DESCRIPTION=RudderStack\'s One trust Interceptor.
+GROUP=com.rudderstack.android.consentfilter
+POM_LICENCE_DIST=repo
+VERSION_NAME=0.0.0
```

Comment:
will this get updated with releashed version?

Reply:
Hopefully. We have "BREAKING CHANGE" in commit message.